### PR TITLE
8.0 partner industry

### PR DESCRIPTION
**Pull request by jt-xx**
Adds an industry sector object and a field 'Industry sector' after the tags on the partner form view.

### CODE REVIEW

File: partner_industry/README.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Industry sector on partner
```
Comment (elicoidal) [inline partner_industry/README.rst:1]: Hi @jt-xx Thanks for your contribution,
I will check mainly formal aspects of the PR.
Please follow the OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
AFAICS, you miss several parts (headers, proper document title, usage section, bug tracker, issues/roadmap if necessary)
We recommend to include a screenshot as well to help new comers

File: partner_industry/README.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline partner_industry/README.rst:25]: https

File: partner_industry/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline partner_industry/__openerp__.py:1]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: partner_industry/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Partner Industry Sector',
+    'version': '1.0',
```
Comment (elicoidal) [inline partner_industry/__openerp__.py]: Check: https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
Versions: 8.0.1.0.0

File: partner_industry/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    'author': "BCIM,Odoo Community Association (OCA)",
+    'maintainer': 'BCIM',
```
Comment (elicoidal) [inline partner_industry/__openerp__.py]: remove key

File: partner_industry/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    'category': 'Sales Management',
+    'complexity': 'easy',
```
Comment (elicoidal) [inline partner_industry/__openerp__.py]: remove key

File: partner_industry/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    'data': [
+        'view/industry.xml',
+        'view/partner.xml',
+        'security/ir.model.access.csv',
+    ],
+    'installable': True,
+    'auto_install': False,
+    'license': 'AGPL-3',
+    'application': False,
+    'images': [],
```
Comment (elicoidal) [inline partner_industry/__openerp__.py:20]: remove empty keys

**Comment by elicoidal** [comment]
Other than minor formal modifications, LGTM

Shouldnot we take the opportunity to add a selection box for Industry classification type (following https://en.wikipedia.org/wiki/Industry_classification and adding "none) and a text field to allow the user to input the formal code?
I dont want to make it complicated with check and constraints (that could be for other additional modules) but it could start an interesting feature.

**Comment by gurneyalex** [comment]
see also discussion on #212

**Comment by yajo** [comment]
It's sad to have to choose in duplicated efforts, but in this case I'd prefer #212 since it's already written in the new api. I invite you to review that other PR and close this.

**Comment by rafaelbn** [comment]
Hi @jt-xx this Pr is duplicated. This functionality already exists. Are you agree with closing this PR? Please let me know, thanks!

**Pull request closed without merging**